packages/perf-benchmarks/rollup-plugins/best-tachometer-shim-runtime.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+/**
```

Comment:
That's fair enough; I mostly saw this as a stepping stone to moving to Tachometer. I wanted to avoid having to rewrite all the tests from scratch, but it definitely wouldn't be a ton of work. Maybe it can be handled in another PR?

Reply:
I thought about this a bit. It occurred to me that if we refactored, we would probably:

- realize we need a framework to avoid duplicating the perf mark/measure and setup/teardown code
- invent an API surface similar to Best
- inject that in the same way I'm injecting it now 😛 

I also kind of like using the shim, as a way of offering other teams an easy migration path from Best to Tachometer, if they choose. What do you think?

Reply:
Correct, we don't need to run the `after` block. Didn't think of that. 😅  We can remove those.

I'm also happy to add an explicit import so that we don't have a hacky global injection.

Reply:
Amusingly, keeping the `after` in actually makes the benchmark run faster overall, because the browser is able to skip doing a style/layout for a large DOM tree. Here's the before-and-after for `benchmark-table/table-create-10k`:

![Screen Shot 2021-07-22 at 11 27 32 AM](https://user-images.githubusercontent.com/283842/126690367-5c6b7715-6a46-4a49-ba35-2cbdb2cd296d.png)

![Screen Shot 2021-07-22 at 11 27 38 AM](https://user-images.githubusercontent.com/283842/126690377-97527ba1-3aaf-4d22-99d3-8581c1584a6b.png)

So I think we should leave the `after` in for now.

There is also an interesting question about whether we should be measuring this style/layout section as well. I would argue that we shouldn't for these benchmarks, but that we should probably create a dedicated benchmark for synthetic shadow styles, and in that case we should measure the style/layout cost.